ios/magnet/ScannerBle.swift

```diff
@@ -9,19 +9,18 @@
 import Foundation
 
 @objc(ScannerBle) class ScannerBle: NSObject, BeaconScannerDelegate {
-  
   var bridge: RCTBridge!
   var scanner: BeaconScanner!
   
   @objc func start() -> Void {
-    NSLog("starting scanner");
-    self.scanner = BeaconScanner();
+    print("starting scanner")
```

Comment:
I found `NSLog()` threw errors when trying to interpolate particular values sometimes. Emily told me to `print()`.